app/controllers/concerns/sufia/files_controller_behavior.rb

```diff
@@ -48,7 +48,7 @@ module FilesControllerBehavior
 
     # routed to /files/new
     def new
-      @batch_id = Batch.create.id
+      @batch_id = Sufia::IdService.mint
```

Comment:
Hmmm, shouldn't `Sufia::IdService.mint` be already getting called behind the scenes?